README.md

```diff
@@ -4,6 +4,18 @@ Command `apcupsd_exporter` provides a Prometheus exporter for the
 [apcupsd](http://www.apcupsd.org/) Network Information Server (NIS). MIT
 Licensed.
 
+## Installation
+
+Make sure that Go package is installed to build this package
+```
+sudo apt install golang
```

Comment:
Not every distribution uses `apt` for package management.

Reply:
But I think it would be good starting point. As there's a mention that some package is required, rather than just listing every single one, few of the most used ones could be listed.

Reply:
So at least mention that this is an example for some specific distro and not a universal command?